charts/kuberpult/Earthfile

```diff
@@ -88,12 +101,20 @@ test-helm:
     LOCALLY
     ARG VERSION=$(git describe --always --long --tags || echo 0.0.1)
 
-    FROM +deps
+    FROM +godeps
     WORKDIR /kp
 
     COPY (+chart-tarball/kuberpult-*.tgz --VERSION=$VERSION) . 
     
+
     RUN helm lint kuberpult-*.tgz --set git.url=test --set ingress.domainName=kuberpult.example.com
+    RUN pwd
+    COPY tests ./tests
+    COPY values.yaml .
+    COPY templates templates
+    COPY Chart.yaml .
+
+    RUN helm dependency build && cd tests && go mod init example.com/test-helm && go mod tidy && go test
```

Comment:
why do we need `example.com/test-helm` here?

Reply:
@ahmed-nour-fdc took a look at the earthly file and didn't see any problem with it, but maybe we missed something?

Reply:
let's leave it as it is for now.